clients/java/client/src/main/java/org/camunda/bpm/client/impl/RequestExecutor.java

```diff
@@ -57,6 +57,7 @@
   protected HttpClient httpClient;
   protected ObjectMapper objectMapper;
 
+
```

Comment:
I think you can revert all changes of this class, so the commit for the PR is clean in the end.

Reply:
Yes, you are right. reverted 
thanks